tests/unit/test_admin.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import absolute_import, unicode_literals
+
+from datetime import datetime
+from unittest import TestCase
+from mock import Mock
+
+from freezegun import freeze_time
+
+from django_celery_monitor.admin import colored_state, node_state, eta,\
+    tstamp, name
+
+
+class ColoredStateTest(TestCase):
```

Comment:
Please move all the tests under a common TestCase class, e.g. `AdminDisplayFieldTests` and use proper unittest assert methods (`self.assertEqual` etc) instead of the `assert` statement.